test/python/circuit/test_circuit_load_from_qpy.py

```diff
@@ -302,7 +303,13 @@ def test_parameter_expression(self):
 
     def test_string_parameter(self):
         """Test a PauliGate instruction that has string parameters."""
-        circ = (X ^ Y ^ Z).to_circuit_op().to_circuit()
+        with warnings.catch_warnings(record=True) as caught_warnings:
+            warnings.filterwarnings(
+                "always",
+                category=DeprecationWarning,
+            )
+            circ = (X ^ Y ^ Z).to_circuit_op().to_circuit()
```

Comment:
Most of the non-algorithm tests, like this one, use opflow but don't really need it, we could just create the circuit from scratch.